rest-services/client/src/main/java/com/_4point/aem/docservices/rest_services/client/forms/RestServicesFormsServiceAdapter.java

```diff
@@ -66,8 +65,47 @@ private RestServicesFormsServiceAdapter(WebTarget target, Supplier<String> corre
 
 	@Override
 	public Document exportData(Document pdfOrXdp, DataFormat dataFormat) throws FormsServiceException {
-		throw new UnsupportedOperationException("exportData has not been implemented yet.");
+		if (pdfOrXdp == null &&dataFormat == null) {
+			throw new FormsServiceException("Internal Error, must provide document and its dataformat");
+		}
+		WebTarget exportDataTarget = baseTarget.path(EXPORT_DATA_PATH);
+		try (final FormDataMultiPart multipart = new FormDataMultiPart()) {
+			multipart.field("pdforxdp", pdfOrXdp.getInputStream(),APPLICATION_PDF)
+					 .field("dataformat", MediaType.APPLICATION_XML);
+			

```

Comment:
See my comment below on the dataformat parameter.  I think this should be `dataFormat.name()`.